api/app/src/test/kotlin/packit/integration/controllers/RoleControllerTest.kt

```diff
@@ -1,9 +1,8 @@
 package packit.integration.controllers
```

Comment:
I got a bit carried away tidying this file. In particular I removed all the manual calls to jacksonObjectMapper and instead use the dtos to deserialize the objects more conveniently.